ts/store/reducers/preferences.ts

```diff
@@ -1,14 +1,22 @@
 import { isActionOf } from "typesafe-actions";
 
+import { preferenceFingerprintIsEnabledSave } from "../actions/preferences";
 import { preferencesLanguagesLoadSuccess } from "../actions/preferences";
 import { Action } from "../actions/types";
+import { GlobalState } from "./types";
 
 export type PreferencesState = Readonly<{
   languages: ReadonlyArray<string> | undefined;
+  isFingerprintEnabled: boolean | undefined;
 }>;
 
+export const isFingerprintEnabledSelector = (
```

Comment:
it doesn't look like this is used anywhere

Reply:
removed